firefox-ios/Client/Frontend/Browser/MainMenu/Views/MainMenuViewController.swift

```diff
@@ -25,6 +25,7 @@ class MainMenuViewController: UIViewController,
     typealias SubscriberStateType = MainMenuState
 
     // MARK: - UI/UX elements
+    private lazy var menuRedesignContent: MenuRedesignMainView = .build()
```

Comment:
Can you add documentation here to know that we have two versions of the menu based on `isMenuRedesign` feature flag